server/chat-plugins/help.ts

```diff
@@ -144,6 +139,14 @@ export class HelpResponder {
 	}
 	match(question: string, faq: string) {
 		if (!this.data.pairs[faq]) this.data.pairs[faq] = [];
+		const room = this.getRoom();
+		if (!room) return;
+		// mostly for the purpose of tests
```

Comment:
?

Reply:
this is because Peach suggested a better way to handle deleting data tied to RFAQs, but the problem is tests fail since I can't edit the roomfaq object in tests in the way it'd need to be edited, nor can I rely on just being able to make the data, so I had to do some fuckery to try to get around it.

Reply:
...I have no clue how this stuff all works so I guess I'll just accept this once the tests pass.